code/game/objects/items/tanks/tanks.dm

```diff
@@ -237,8 +237,7 @@
 
 	if(pressure > TANK_FRAGMENT_PRESSURE)
 		if(!istype(src.loc, /obj/item/transfer_valve))
-			message_admins("Explosive tank rupture! Last key to touch the tank was [src.fingerprintslast].")
-			log_game("Explosive tank rupture! Last key to touch the tank was [src.fingerprintslast].")
+			log_bomber(get_mob_by_key(fingerprintslast), "was last key to touch", src, "which ruptured explosively")
```

Comment:
realistically this needs fingerprints of the guy who opened the valve in a can and the guy who ejected it from the can, since the last guy to touch it could very well be a guy who goes "oh fuck a maxcap, better throw it out"